tests/epyccel/test_parallel_epyccel.py

```diff
@@ -0,0 +1,81 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring/
+from mpi4py import MPI
+import pytest
+import numpy as np
+
+from pyccel.epyccel import epyccel
+
+@pytest.fixture(params=[
+    pytest.param('fortran', marks = pytest.mark.fortran),
+    pytest.param('c'      , marks = [pytest.mark.c,
+        pytest.mark.xfail(message='Arrays not implemented in C')])
+    ]
+)
+def language(request):
+    return request.param
```

Comment:
This is not necessary. Pytest finds this automatically in the conftest.py file

Reply:
But I need to say that C fails...

Reply:
Ah ok, sorry. It's a shame that we don't have any tests for this that don't need arrays. But I suppose this should be a non-issue soon